resolwe_bio/processes/expression/featureCounts.yml

```diff
@@ -409,14 +438,33 @@
              |format(alignment.aligned_reads.build, annotation.annotation.build) }}
       {% endif %}
 
+      {% if alignment.assay_type == 'auto' and not alignment.cdna_index %}
+        re-error 'cDNA sequence index must be provided to automatically detect strandedness.'
+      {% endif %}
+
+      {% if alignment.assay_type == 'auto' and alignment.cdna_index %}
+        {% if alignment.aligned_reads.species != alignment.cdna_index.species %}
+          re-error \
+            {{ ('Species of aligned reads (%s) and cDNA index (%s) do not match. ' ~
+              'Please provide aligned reads and cDNA index with the same species.')
+              |format(alignment.aligned_reads.species, alignment.cdna_index.species) }}
+        {% endif %}
+      {% endif %}
+
+      . detect_strandedness.sh \
```

Comment:
The script should be structured in such a way that it can be called without the leading dot.

Reply:
If I `echo $STRANDEDNESS` and save it to a variable within the process, than I cannot use `re-save` / `re-checkrc` commands, since they also print to stdout. Its either leading dot or no error reporting, so I took a lesser evil. Any other suggestion, other than doing it in Python when the time is right?